### ISSUE TITLE
Export current state

### ISSUE THREAD

**Issue by Abbe98** [sonnyp/Playhouse#8]
Currently one can either lookup the HTML/CSS/JS files in the app's data directory, copy those files, and then connect them by manually adding `link` and `script` tags or one can copy paste the panel contents into new files.

It would be an improvement if one could export the current state directly from the application itself. Such an function could either create a HTML file with the CSS and JS injected, a directory or a zip-file.

**Comment by sonnyp** [comment]
Absolutely, but I think the export should be readable by a browser directly – so probably one big HTML file.

Would you like to give it a try? I can take 10 minutes to improve the contribution guide.

**Comment by sonnyp** [comment]
https://github.com/sonnyp/Playhouse/issues/10
